Makefile

```diff
@@ -91,6 +91,11 @@ test-libsql-sqlite: dev-libsql-sqlite test-qe-st
 
```

Comment:
It would be great if we could allow for specifying the runner for `test-qe`:

`DRIVER_ADAPTER=name [RUNNER=napi|wasm make-qe]`

```make
test-qe:
ifndef DRIVER_ADAPTER
	cargo test --package query-engine-tests
else
	@echo "Executing query engine tests with $(DRIVER_ADAPTER) driver adapter"; \
	# Add your actual command for the "test-driver-adapter" task here
	$(MAKE) test-driver-adapter-$(DRIVER_ADAPTER);
endif
```

And this [section of the docs](https://github.com/prisma/prisma-engines/tree/main/query-engine/connector-test-kit-rs#running-tests-through-driver-adapters) was updated accordingly

Reply:
Nevermind, misunderstood your comment. Added.